### PR TITLE
Add Turhost

### PR DESCRIPTION
**Pull request by MesutErdemir**

### CODE REVIEW

**Review by phallobst — CHANGES_REQUESTED** [review]
We still need a logo for this entry, preferably in SVG format.

Also, is there some sort of documentation publicly available? Otherwise, could you provide a screenshot of the 2FA setup process (with any personal information blurred or redacted)?

File: entries/t/turhost.com.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "contact": {
+      "facebook": "turhost",
+      "twitter": "turhost",
+      "linkedin": "turhost",
+      "instagram": "turhost"
+    },
```
Comment (phallobst) [inline entries/t/turhost.com.json:14]: Once there is a `tfa` section, the `contact` section is not used anymore (which does not support `linkedin` or `instagram` anyway).
```suggestion
```

File: entries/t/turhost.com.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "keywords": [
+      "hosting",
+      "server"
```
Comment (phallobst) [inline entries/t/turhost.com.json:17]:
```suggestion
      "domains"
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/t/turhost.com.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "Hosting": {
```
Comment (Carlgo11) [inline entries/t/turhost.com.json:2]:
```suggestion
  "Turhost": {
```

File: entries/t/turhost.com.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "domain": "turhost.com",
+    "url": "https://www.turhost.com/hosting/",
```
Comment (Carlgo11) [inline entries/t/turhost.com.json:4]:
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The `doc` tag needs to be used or screenshots of the 2FA setup process would need to be provided in order for us to validate that 2FA is indeed supported.

Please fix the requested changes within a few days or we'll have to close the PR.

**Pull request closed without merging**